powerauth-push-server/src/main/java/io/getlime/push/controller/rest/PushCampaignController.java

```diff
@@ -273,6 +281,20 @@ public Response deleteUsersFromCampaign(@PathVariable(value = "id") Long id, @Re
         return new Response();
     }
 
+    /**
+     * Find application credentials by PowerAuth app ID.
+     * @param powerAuthAppId PowerAuth App ID.
```

Comment:
Why don`t we use an empty line in javadoc? (I have noticed it at more places in Wultra code) https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#writingdoccomments